src/pages/industries/edge-computing.jsx

```diff
@@ -0,0 +1,112 @@
+  withPerson: true,
+  CTAtext: 'Read The Blog Post',
+  name: 'Tomoya Fujita',
+  role: 'Software Engineer, Sony',
+  url: 'https://kubeedge.io/blog/enable-cilium/',
+};
+
+const bulletSection1 = {
+  heading: 'Deep Visibility for Edge Operations',
+  paragraphs: [
+    "Cilium revolutionizes observability in complex edge environments where traditional monitoring tools fall short. By leveraging Hubble for real-time network flow analysis and Tetragon for security event monitoring, Cilium provides deep security and visibility across distributed edge topologies. Cilium's seamless integration with SIEM and monitoring platforms allows organizations to maintain consistency in their security and observability practices from cloud to edge. ",
+    'With Cilium, operators gain deep, actionable insights into their edge infrastructure, empowering them to quickly identify, troubleshoot, and resolve issues across their entire network, thereby maintaining optimal performance and security in even the most challenging edge computing scenarios.',
+  ],
+  imageSrc: EdgeImage1,
+  imageAlt: 'cilium big tcp stats',
+  imageRight: false,
+};
+
+const bulletSection2 = {
+  heading: 'Optimized for Resource-Constrained Environments',
+  paragraphs: [
+    'Edge computing often means working with limited resources. Cilium is designed to be lightweight and efficient, leveraging cutting-edge eBPF technology to deliver unparalleled performance and efficiency in resource-constrained edge environments. Cilium is optimized for both x86 and ARM-based architectures. By implementing load balancing directly in XDP, Cilium achieves remarkable resource efficiency.',
+    'With Cilium, businesses can maximize the potential of their edge infrastructure, handling more traffic with fewer resources. This efficiency translates to reduced hardware costs, improved energy consumption, and the ability to deploy more powerful edge applications on existing hardware.',
+  ],
+  imageSrc: EdgeImage2,
+  imageAlt: 'hetzner cilium test illustration',
+};
+
+const bulletSection3 = {
+  heading: 'Seamless Connectivity  for Edge Infrastructures',
+  paragraphs: [
+    "Cilium's ClusterMesh feature revolutionizes edge computing by seamlessly extending Cilium's powerful networking and security capabilities across multiple clusters. It enables secure, high-performance connectivity between edge locations, data centers, and cloud resources without complex VPN setups. ClusterMesh facilitates global service discovery, unified security policies, and intelligent cross-cluster load balancing, ensuring consistent protection and optimal performance across your entire edge infrastructure.",
+    'This unified approach significantly reduces operational complexity while enhancing the resilience and efficiency of distributed edge applications.',
+  ],
+  imageSrc: EdgeImage3,
+  imageAlt: 'hetzner cilium test illustration',
+};
+
+const cloudTalks = [
+  {
```

Comment:
Can you also add the Roche talk? https://www.youtube.com/watch?v=kC8efabCH6s

maybe cut the arm64 one too